source/_components/sensor.etherrain.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "EtherRain Sensor"
```

Comment:
Is the sensor necessary? It's as easy to monitor if a sensor is on as it is to monitor if a switch is on in HA.